tests/x11regressions/gnote/gnote_first_run.pm

```diff
@@ -14,13 +14,25 @@
 use base "x11regressiontest";
 use strict;
 use testapi;
+use utils;
 
 sub run {
+    if (is_tumbleweed) {
+        select_console('root-console');
+        pkcon_quit;
+        zypper_call('in gnote');
+        select_console('x11');
+    }
     x11_start_program("gnote");
     assert_screen "gnote-first-launched", 10;
 
     send_key "ctrl-w";
 }
 
+# add milestone flag to save gnote installation in lastgood vm snapshot
+sub test_flags {
+    return {milestone => 1} if (is_tumbleweed);
```

Comment:
Sorry, this line caused a new issue, I will return nothing on non tumbleweed scenarios, I filed a poo for this:
https://progress.opensuse.org/issues/23840.

I will fix it asap.

Reply:
fix: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3508